apps/api/src/lib/coscrad-query-language/aql/compile-aql-filter-block.ts

```diff
@@ -508,6 +512,78 @@ const compileSimpleFilterCondition = (
         };
     }
 
+    if (operator === CoscradBooleanOperator.USER_CAN) {
```

Comment:
Let's be careful that there are no security implications of this. It shouldn't be possible for a user to inject a different user onto the query. It's probably safest to
- remove this operator
- ensure that the `accessControlList` is a forbidden property

Reply:
This has been done.